frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -0,0 +1,67 @@
+# Contributing Data
+
+Cellxgene supports a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you have single cell data that meet our [requirements](#dataset-requirements), please reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com).
+
+## Publishing Process
+
+The process for submission to the portal is:
+- You confirm we can support your submission by reaching out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) with a description of the data that you'd like to contribute.
+- We confirm that we will accept your data.
+- You prepare your data according to our submission [requirements](#dataset-requirements) and send us your files.
+- We upload to a private collection where you can review.
+- We publish the data when you tell us to.
```

Comment:
I suggest adding "You prepare revised data and send us your revised files, as needed." before the publish step.

Reply:
```suggestion
- You prepare revised data and send us your revised files, as needed.
- We publish the data when you tell us to.
```